flow360/component/simulation/services.py

```diff
@@ -207,7 +210,6 @@ def get_default_params(
 
 def validate_model(
```

Comment:
Lets starting from this function enforce the "too-many-positional-arguments" linting.
```suggestion
def validate_model(*,
```

Reply:
OK! I just add asterisk to validate_model and generate_process_json functions, which are both updated to fix the unit_system_name.